### ISSUE TITLE
[Feedback]: Telegram Notification

### ISSUE THREAD

**Issue by JZ-SmartThings** [netdata/learn#1991]
### Doc Page link

https://learn.netdata.cloud/docs/alerts-&-notifications/notifications/agent-dispatched-notifications/telegram

### Description

Please remove the minus sign from the Chat ID numbers. It didn't work at all for me with HTTP 400 error until I just removed the minus sign. Then it worked flawlessly. Thank you!

**Comment by Ancairon** [comment]
Hey @JZ-SmartThings what type of chat you got? In our docs it mentions `Group IDs start with a hyphen, supergroup IDs start with -100`

**Comment by JZ-SmartThings** [comment]
In the BotFather I just did /newbot then joined it, then wrote an initial message so that it shows up at this URL:

https://api.telegram.org/botXXX:YYY/getUpdates

After getting that, it's just a number without any dashes. It would be the chat.id field of the json. Once I took out the dash which I obviously saw in your instructions, it finally worked. With a dash it did not, said couldn't find chat ID.

Thanks!
